Dockerfile

```diff
@@ -1,19 +1,17 @@
-FROM  quay.io/wtsicgp/dockstore-cgpmap:3.1.4 as builder
+FROM quay.io/wtsicgp/pcap-core:5.2.0 as builder
 
 USER root
 
 # ALL tool versions used by opt-build.sh
 ENV VER_CGPVCF="v2.2.1"
 ENV VER_VCFTOOLS="0.1.16"
-#ENV VER_BIODBHTS="2.10"
-#ENV VER_HTSLIB="1.7"
-ENV VER_ALLELECOUNT="v4.0.0"
+ENV VER_ALLELECOUNT="4.1.0"
```

Comment:
need to be more careful of changing the version format of releases.  I see that alleleCount has been changed from having a `v` prefix.